apps/projects/app/components/Content/IssueDetail/StatusCard.js

```diff
@@ -7,18 +8,34 @@ import {
   Text,
   Button,
 } from '@aragon/ui'
-//import { usePanelManagement } from '../../Panel'
 import { issueShape } from '../../../utils/shapes.js'
 import { usePanelManagement } from '../../Panel'
 import { BOUNTY_STATUS_LONG } from '../../../utils/bounty-status'
 
+const Action = ({ panel, caption, issue }) => (
+  <EventButton
+    mode="outline"
+    wide
+    onClick={() => panel(issue)}
+  >
+    <Text weight="bold">
+      {caption}
+    </Text>
+  </EventButton>
+)
+Action.propTypes = {
+  panel: PropTypes.func.isRequired,
+  caption: PropTypes.string.isRequired,
+  issue: issueShape,
+}
+
 const StatusCard = ({ issue }) => {
   const theme = useTheme()
-  const { requestAssignment } = usePanelManagement()
-
+  const { submitWork, requestAssignment, reviewApplication, reviewWork } = usePanelManagement()
+  
   const determineStatus = (workStatus, balance) =>
     Number(balance) === 0 ? BOUNTY_STATUS_LONG['fulfilled'] : BOUNTY_STATUS_LONG[workStatus]
-
+  
```

Comment:
Please configure your editor to remove trailing whitespace.

Reply:
Trim Trailing. Done!